src/core/lib/surface/channel.h

```diff
@@ -158,7 +156,7 @@ class Channel : public RefCounted<Channel>,
   }
 
   grpc_event_engine::experimental::EventEngine* event_engine() const {
-    return event_engine_.get();
+    return channel_stack_->EventEngine();
```

Comment:
(no change required)

Probably ok for now, we might want to cache the pointer on this object at some point though (thinking about number of pointer chases on common per-call paths)

CC @markdroth

Reply:
I was thinking about that, but wouln't any per-call paths get this pointer from the promise context instead?  I would expect that only control-plane code would actually wind up using this method.